docs/azure/sdk/authentication/create-token-credentials-from-configuration.md

```diff
@@ -118,10 +129,15 @@ Add the following configuration values to your _appsettings.json_ file to create
 {
     "tenantId":  "<tenantId>",
     "clientId":  "<clientId>",
-    "clientCertificate": "<clientCertificate>"
+    "clientCertificate": "<clientCertificate>",
+    "clientCertificateStoreLocation": "<clientCertificateStoreLocation>",
+    "additionallyAllowedTenants": "<tenant-ids-separated-by-semicolon>"
 }
 ```
 
+> [!NOTE]
+> The `clientCertificateStoreLocation` and `additionallyAllowedTenants` values are optional.
```

Comment:
It may be worth adding that if no `clientCertificateStoreLocation` is specified, the default `CurrentUser` is used (reference of cert store location values is [here](https://learn.microsoft.com/dotnet/api/system.security.cryptography.x509certificates.storelocation))